descartes_planner/include/descartes_planner/planning_graph.h

```diff
@@ -125,6 +123,14 @@ class PlanningGraph
 
   descartes_core::RobotModelConstPtr getRobotModel();
 
+
+  CartesianMap getCartesianMap() const;
```

Comment:
This is the function interface that this function has always had; I just moved it to be with the other helpers.

It doesn't return a const reference because the variable it seeks to return is a raw pointer. In the case that the user calls this function before the pointer has been allocated, we return an empty map.

In short, I haven't fixed this bit of ugliness in the code quite yet.